lib/components/viewers/route-viewer.js

```diff
@@ -247,7 +255,7 @@ const RouteDetails = styled.div`
 `
 
 class RouteRow extends PureComponent {
-  static contextType = ComponentContext
+  static contextType = ComponentContext;
```

Comment:
Remove semicolon

Reply:
Done in 59bdf8f495ea26a3a31f7b48ad10a767b0ca8ef7